### PR TITLE
event docs

### PR DESCRIPTION
**Pull request by atotic**

### CODE REVIEW

File: google-feeds.html [hunk 1]
```diff
@@ -30,26 +30,30 @@
...
-
-     Fired when feed query fails
-     @param {string} status status
-     @event 'google-feeds-queryerror'
-
-     Fired when feed query succeeds
-     @param {object} entries query results
-     @event 'google-feeds-queryresponse'
-    */
+     * Fired when feed has been loaded
```
Comment (ebidel) [inline google-feeds.html:33]: These all need to separate comment blocks? That's a weird requirement.

Comment (atotic) [inline google-feeds.html:33]: Yep. The way parser works is that it uses all free text within the block as description, and all @tags as properties of that comment. 

Aleks
-- sent from my phone

**Comment by ebidel** [comment]
Can you merge with the latest 0.8-preview so this is mergeable?

**Comment by atotic** [comment]
done!

**Pull request merged**